docs/index.html

```diff
@@ -2,69 +2,191 @@
 layout: default
 title: eBay Skin
 ds: 6
+ds_map:
+    alert-dialog:
+        name: alert-dialog
+        version: 2.1
+    avatar:
+        name: avatar
+        version: 1.0
+    badge:
+        name: badge
+        version: 1.2
+    breadcrumbs:
+        name: breadcrumb
+        version: 1.2
+    button:
+        group: button
+        name: button
+        version: 1.6
+    carousel:
+        name: carousel
+        version: 1.1
+    calendar:
+        name: calendar
+        version: 1.0
+    checkbox:
+        name: checkbox
+        version: 2.0
+    chip:
+        version: 1.0.0
+    chips-combobox:
```

Comment:
Was there going to be a second pass to go through these dependencies? Perhaps DS docs are unaware of the modules Skin is utilizing inside other modules?

For `chips-combobox` specifically, we are using `chip` and `combobox`.